packages/common.vm/tools/vm.common/vm.common.psm1

```diff
@@ -242,15 +241,11 @@ function VM-Install-Raw-GitHub-Repo {
         }
 
         if ($powershellCommand) {
-            $executableArgs = "-ExecutionPolicy Bypass -NoExit -Command $powershellCommand"
             $powershellPath = Join-Path "${Env:WinDir}\system32\WindowsPowerShell\v1.0" "powershell.exe" -Resolve
-            $shortcut = Join-Path $shortcutDir "$toolName.lnk"
-            Install-ChocolateyShortcut -shortcutFilePath $shortcut -targetPath $powershellPath -Arguments $executableArgs -WorkingDirectory $toolDir -IconLocation $powershell
+            VM-Install-Shortcut -toolName $toolName -category $category -executablePath $powershellPath -arguments $powershellCommand -executableDir $toolDir -powershell $true
```

Comment:
`executablePath` is not used for powerShell inside `VM-Install-Shortcut`. I think we need to make this argument optional to simplify the call for PowerShell and remove the need to assign `$powershellPath`

Reply:
I think this was to make the shortcut icon show powershell, which seems a bit weird to do it this way looking back at it.

I can change it instead such that if `$powershell` is set to `$true`, that `$IconLocation = Join-Path "${Env:WinDir}\system32\WindowsPowerShell\v1.0" "powershell.exe" -Resolve` on ~line 297 just after setting the `$executableCmd` and `$executableArgs`

Reply:
we can change `$executablePath` to optional